ydb/library/yql/sql/v1/node.h

```diff
@@ -1152,11 +1152,39 @@ namespace NSQLTranslationV1 {
         TNodePtr Compression;
     };
 
+    struct TVectorIndexSettings {
+        enum class EDistance {
+              Cosine        /* "cosine" */
+            , Manhattan     /* "manhattan" */
+            , Euclidean     /* "euclidean" */
+        };
+
+        enum class ESimilarity {
+              Cosine        /* "cosine" */
+            , InnerProduct  /* "inner_product" */
+        };
+
+        enum class EVectorType {
+              Float         /* "float" */
+            , Uint8         /* "uint8" */
+            , Int8          /* "int8" */
+            , Bit           /* "bit" */
+        };
```

Comment:
We can keep enum style same in the file:
```cpp
enum class Enum {
  First,
  Second,
  Third,
};
```
Also I don't understand purpose of this comments, they're kind of same with enum members